string topic
---
sensor_msgs/CompressedImage result
